source/frontend/shared/dAPIResponses.js

```diff
@@ -0,0 +1,21 @@
+const { DiscordjsErrorCodes } = require("discord.js");
+
+/** @param {(error) => boolean} ignoreThese */
+function butIgnoreCertainErrors(ignoreThese) {
+	return (error) => {
+		if (!ignoreThese(error)) {
+			console.error(error);
+		}
+	}
+}
```

Comment:
Allowing this to take in any number of functions would be a minor but useful quality of life for composing multiple errors into one check in the future.
```suggestion
function butIgnoreCertainErrors(...ignoreThese) {
	return (error) => {
		for (const ignoreThis of ignoreThese) {
			if (ignoreThis(error)) {
				return;
			}
		}
		console.error(error);
	}
}
```